src/Kunstmaan/SeoBundle/Controller/Admin/SettingsController.php

```diff
@@ -68,6 +51,11 @@ public function robotsSettingsAction(Request $request)
             }
         }
 
+        if ($robot->getRobotsTxt() === $default) {
```

Comment:
I think this is not the correct check. When someone saves the default text, there is an entry in the database, but `$robot->getRobotsTxt() === $default`is still true...